Liteflow file (#244)
liteflow-labs/liteflow-js: packages/cli/CHANGELOG.md

#### Breaking Changes

> - [#244](https://github.com/liteflow-labs/liteflow-js/pull/244) Remove argument in `deploy` and `dev` command. To deploy and/or dev, the command needs to run the command in the directory of the project.

Reviewer: ```suggestion
- [#244](https://github.com/liteflow-labs/liteflow-js/pull/244) Remove argument path in `deploy` and `dev` commands. You need to run the commands in the directory of your project.
```

#### Improvements

> - [#](https://github.com/liteflow-labs/liteflow-js/pull/) Support new liteflow.yml file.

Author: ```suggestion
- [#244](https://github.com/liteflow-labs/liteflow-js/pull/244) Support new liteflow.yml file.
```
